app/views/teacher_interface/uploads/new.html.erb

```diff
@@ -30,9 +30,9 @@
     <h1 class="govuk-heading-l">Upload the Medium of instruction (MOI) from your university degree</h1>
     <p class="govuk-body">This document must confirm that the primary language used to teach this degree course was English.</p>
   <% elsif @document.english_language_proficiency? %>
-    <h1 class="govuk-heading-l">Upload evidence of English Proficiency test</h1>
-    <p class="govuk-body">This document must confirm that you have an English Language Proficiency Test from an approved provider at level B2 of the Common European Framework of Reference for Languages (CEFR) scale.</p>
-    <p class="govuk-body">The test must be on the list of approved English language tests and awarded within the last 2 years before the date of your application.</p>
+    <h1 class="govuk-heading-l">Upload evidence of English language proficiency test</h1>
+    <p class="govuk-body">This document must confirm that you have an English Language proficiency test from an approved provider at level B2 of the Common European Framework of Reference for Languages (CEFR) scale.</p>
```

Comment:
Should line 34 'Language' be lower case? Or should line 35 be upper case?

Reply:
I think it's meant to be _English language_ if referring to the language and _English Language_ if referring to the qualification

Reply:
I think both are referring to the test/qualification so should be upper case on line 34?

Reply:
I'll check with Wes...!

Reply:
Turns out we we all wrong, I've amended the commit with the correct words.

Reply:
Wes has confirmed that only `English` should have a capital letter though.